src/OrbitGl/TrackManager.cpp

```diff
@@ -66,97 +66,86 @@ std::vector<FrameTrack*> TrackManager::GetFrameTracks() const {
 }
 
 void TrackManager::SortTracks() {
-  if (!app_->IsCapturing() && !sorted_tracks_.empty() && !sorting_invalidated_ &&
-      !visible_track_list_needs_update_)
-    return;
-
-  // Reorder threads if sorting isn't valid or once per second when capturing
-  if (sorting_invalidated_ ||
-      (app_->IsCapturing() && last_thread_reorder_.ElapsedMillis() > 1000.0)) {
-    std::lock_guard<std::recursive_mutex> lock(mutex_);
-    // Gather all tracks regardless of the process in sorted order
-    std::vector<Track*> all_processes_sorted_tracks;
+  std::lock_guard<std::recursive_mutex> lock(mutex_);
```

Comment:
Can you use this opportunity to add the [GUARDED_BY](https://abseil.io/docs/cpp/guides/synchronization#annotation-guidelines) annotation to the members that you know are protected by the mutex?

Reply:
Added to the correspondent bug: http://b/177200020. I feel we should revise it properly, and not only add what I think that are being protected.